debian/copyright

```diff
@@ -0,0 +1,31 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: tvb-library
+Upstream-Contact: TVB TEAM <tvb.admin@thevirtualbrain.org>
+Source: https://github.com/the-virtual-brain/tvb-library
```

Comment:
Again tvb-library in the URL. It should be the current package!!!!
Please check all similar URLs